shoop/admin/templates/shoop/admin/orders/detail.jinja

```diff
@@ -126,6 +126,32 @@
                             </div>
                         </div>
                     {% endcall %}
+
+                    {% set payments=order.payments.all() %}
+                    {% if payments %}
+                        {% call content_block(_("Payments"), "fa-dollar") %}
+                            <table class="table table-striped table-bordered">
```

Comment:
Since we are using table here, could this be done the same way as the tab "order contents" where the table is hidden in mobile and list-group is used instead?